.github/workflows/webpack-jumpstart.yml

```diff
@@ -25,7 +25,10 @@ jobs:
              git fetch --no-tags origin ${BASE_SHA}:base ${HEAD_SHA}:head
              git checkout --detach base
              git merge --no-edit head
-             pkg/build webpack-jumpstart.tar -j$(nproc)")"
+             git rev-parse HEAD^{tree} > tree
+             tools/node-modules make_package_lock_json
```

Comment:
Note: This line is temporary, to make this workflow-changing PR as small as possible. My next PR will teach webpack-make.js to check out n_m/ by itself, and drop this again.